### ISSUE TITLE
Visualizations: in a Box plot of participant repeated measures, only community-level vars can be selected for the overlay

### ISSUE THREAD

**Issue by danicahelb** [VEuPathDB/EdaDataService#106]
using the WASH-b Kenya study, I made a box plot of LAZ (participant repeated measure entity) for each study arm (community entity). I'd like to overlay this by either "study timepoint" or "diarrhea in last 7 days" (both under participant repeated measure entity) but I can only select overlay variables in the community entity

![image](https://user-images.githubusercontent.com/25450900/135361826-6c093b9c-7a4f-4820-9663-6cf6f566999e.png)

**Comment by d-callan** [comment]
currently this is by design. weve implemented variable constraints in boxplot so that the strata vars have to be of the same or parent entity of the xaxis term. i suppose we can revisit that if wed like, but idk if its a good idea for phase 1 given how little time is left. it seems like what youre proposing is that rather than have some dependency order like yaxis -> xaxis -> overlay -> facet, we might have only the var that sets the entity space of the plot specified (in this case yaxis) and ALL other vars must simply be of the same or parent entity as that one.

**Comment by danicahelb** [comment]
@d-callan i agree that it would make more sense to only specify the var that sets the entity space of the plot and ALL other vars must simply be of the same or parent entity as that one, but this can wait until phase 2.

In the 10/1 UX meeting we decided that we will not change the variable constraints as described above for Phase 1.

can you write up the logic for variable constraints and we will add this information to the “Learn more” link for Phase 1?
![image](https://user-images.githubusercontent.com/25450900/135703967-95c525ac-c06f-4c82-be8e-65f70e07f8c6.png)
